js/ui/components/clubhouse.js

```diff
@@ -369,6 +369,10 @@ class ClubhouseNotificationBanner extends MessageTray.NotificationBanner {
         if (!this._inLastPage())
             return;
 
+        // If we paginated, remove the next button before adding others.
```

Comment:
I was wondering if I should comment on the `child-*` code to suggest updating it whenever a child is added or removed, but I thought it could always be added later if it was necessary. However, after reading this, it seems necessary :smile: In particular, because it seems likely we will continue to accumulate workarounds to make sure all the buttons are added in the right order, if we need to add any features to the buttons later.

Reply:
Yes, this is a workaround for the missing `nth-child` workaround :(

Reply:
But can we give the workaround a bit more fidelity, by making it update the CSS classes when buttons are added/removed so that we don't have this requirement of never adding/removing buttons once the CSS classes are assigned?